test/e2e/pages/allocation-journey.js

```diff
@@ -0,0 +1,52 @@
+import { ClientFunction, t } from 'testcafe'
+
+import AllocationCancelPage from './allocation-cancel'
+import AllocationCriteriaPage from './allocation-criteria'
+import AllocationDetailsPage from './allocation-details'
+import Page from './page'
+
+const allocationDetailsPage = new AllocationDetailsPage()
+const allocationCriteriaPage = new AllocationCriteriaPage()
+const allocationCancelPage = new AllocationCancelPage()
+
+class AllocationJourney extends Page {
+  constructor() {
+    super()
+    this.allocationCancelPage = allocationCancelPage
+    this.allocationCriteriaPage = allocationCriteriaPage
+    this.allocationDetailsPage = allocationDetailsPage
+  }
+
+  async createAllocation() {
+    await allocationDetailsPage.fill()
+    await this.submitForm()
+    await allocationCriteriaPage.fill()
+    await this.submitForm()
+    return t
+      .expect(this.getCurrentUrl())
+      .match(/\/allocation\/[\w]{8}(-[\w]{4}){3}-[\w]{12}\/confirmation$/)
+  }
+
+  async triggerValidationOnAllocationDetailsPage() {
+    await this.submitForm()
```

Comment:
Doesn't feel like we need to have this method as it's calling a method that already exists on the base Page object to submit the form.

Instead of 

```javascript
await allocationJourney.triggerValidationOnAllocationDetailsPage()
```

you can just use

```javascript
await page.submitForm()
```